src/main/java/medicaldoctor/Program.java

```diff
@@ -1,18 +1,37 @@
 package medicaldoctor;
 
+import com.sun.glass.ui.Screen;
 import javafx.application.Application;
 import javafx.fxml.FXMLLoader;
+import javafx.scene.Group;
 import javafx.scene.Parent;
 import javafx.scene.Scene;
 import javafx.stage.Stage;
 import medicaldoctor.core.DatabaseScope;
 
 public class Program extends Application {
+    
+    //Use better methold than this
+    public static String ScreenLoginName = "Login";
```

Comment:
java uses camel case, not pascal case, also these are constants so they can have the `final modifier`, which changes the naming convention to CAPS_WITH_UNDERSCORE.

```suggestion
    public static final String SCREEN_LOGIN_NAME = "Login";
```